pandas/tests/indexing/test_scalar.py

```diff
@@ -214,6 +220,8 @@ def test_mixed_index_at_iat_loc_iloc_dataframe(self):
         with pytest.raises(KeyError, match="^3$"):
             df.loc[0, 3]
 
+    # TODO(CoW) fix at/iat setting on a DataFrame
+    @td.skip_array_manager_invalid_test
```

Comment:
why is it invalid?  the TODO makes me think it should be an xfail (seem with L54)

Reply:
Yeah, that should have been an xfail, but it also already seems fixed now, so removed it.